examples/role.ts

```diff
@@ -0,0 +1,23 @@
+import { createClient } from '@clickhouse/client' // or '@clickhouse/client-web'
+
+void (async () => {
+  const client = createClient({
+    role: 'role_name_1',
+  })
+
+  // with a role defined in the client configuration, all queries will use the specified role
+  await client.command({
+    query: `SELECT * FROM SECURED_TABLE`,
+  })
```

Comment:
This should be a `query` call; now it is a misuse in an example, we want to avoid that

Reply:
Updated all SELECT queries to use `.query` instead of `.command`